spec/javascripts/selectors/columns_spec.js

```diff
@@ -0,0 +1,82 @@
+import storyFactory from '../support/factories/storyFactory';
+import { getColumns } from '../../../app/assets/javascripts/selectors/columns';
+import actionTypes from '../../../app/assets/javascripts/actions/actionTypes';
+import moment from 'moment';
+
+describe('Columns Selector', () => {
+  const unscheduledStory = storyFactory({ state: 'unscheduled' });
+  const startedStory = storyFactory({ state: 'started' });
+  const deliveredStory = storyFactory({ state: 'delivered' });
+
+  const currentSprintDate = moment();
+  const previousSprintDate = moment().subtract(1, 'weeks');
+
+  const storiesArray = [unscheduledStory, startedStory, deliveredStory];
+
+  describe('CHILLY_BIN', () => {
+    it('return only unschedule stories', () => {
+      const chillyBinStories = getColumns({
+        column: actionTypes.COLUMN_CHILLY_BIN,
+        stories: storiesArray
+      });
+
+      expect(chillyBinStories).toContain(unscheduledStory);
+      expect(chillyBinStories).not.toContain(startedStory);
+      expect(chillyBinStories).not.toContain(deliveredStory);
+    });
+  });
+
+  describe('BACKLOG', () => {
+    const storyAcceptedCurrentSprint = storyFactory({ state: 'accepted', acceptedAt: currentSprintDate });
+
+    const project = {
+      iterationLength: 1,
+      iterationStartDay: 1,
+      startDate: currentSprintDate.format()
+    };
+
+    it('return story accepted in current sprint', () => {
+      storiesArray.push(storyAcceptedCurrentSprint);
```

Comment:
You're mutating shared state here as well, we could use the same approach for the other comment. Using `let`  + `beforeEach`